### PR TITLE
Caribou dSiPM Updates

### PR DESCRIPTION
**Pull request by simonspa**

### CODE REVIEW

**Comment by simonspa** [comment]
(cc) @ffeindt

File: user/tlu/module/src/TluRawEvent2StdEventConverter.cc [hunk 1]
```diff
@@ -131,6 +131,14 @@ bool TluRawEvent2StdEventConverter::Converting(eudaq::EventSPC d1, eudaq::Standa
   d2->SetTimeEnd(d1->GetTimestampEnd() * 1000);
   d2->SetTimestamp(ts, d1->GetTimestampEnd(), d1->IsFlagTimestamp());
 
+  // print time stamps to have a look
```
Comment (lhuth) [inline user/tlu/module/src/TluRawEvent2StdEventConverter.cc:134]: Guess that's a leftover from debugging? :)

Comment (ffeindt) [reply user/tlu/module/src/TluRawEvent2StdEventConverter.cc:134]: yeth, not sure how this ended up here ;)

**Review by lhuth — APPROVED** [review]
🚀

**Pull request merged**